### PR TITLE
Update sentry

### PR DESCRIPTION
**Pull request by Noah-Houghton**
Update the SentryError handler to use `sentry-sdk` instead of `raven`

### CODE REVIEW

**Comment by biblicabeebli** [comment]
I'm pretty thrilled to see this, sorry I was never able to get around to it.

**Comment by biblicabeebli** [comment]
FYI on a particular pain point.
In Raven the default `transport` value for the sentry client is **asynchronous** - in the sense that it was non-blocking and delegated to some background thread.  The documentation for the new client makes no statement about this behavioral detail.
https://docs.sentry.io/platforms/python/configuration/options/

If you had a sentry client doing its thing, sending error reports on a background thread, the main thread exiting could cause you to lose those error reports.  (why? no clue. the Raven devs designed it that way.)  It would probably be a good idea to check the new default and determine if there has been a behavior change, if it changes include a comment in the pipy update.

File: cronutils/error_handler.py [hunk 1]
```diff
@@ -113,10 +113,10 @@ def __init__(self, sentry_dsn, descriptor=None, data_limit=100, sentry_client_kw
             sentry_report_limit=0
     ):
         if sentry_client_kwargs:
-            self.sentry_client = SentryClient(dsn=sentry_dsn, **sentry_client_kwargs)
+            sentry_sdk.init(sentry_dsn, **sentry_client_kwargs)
         else:
-            self.sentry_client = SentryClient(dsn=sentry_dsn)
-            
+            sentry_sdk.init(sentry_dsn)
```
Comment (zags) [inline cronutils/error_handler.py:118]: sentry_sdk may have global config.  See if we can make the DSN optional and skip calling `init` if no params are passed.

**Comment by biblicabeebli** [comment]
iiiiis this ever gonna get merged?

**Review by zags — APPROVED** [review]

**Comment by sesterki** [comment]
@biblicabeebli It has been merged. Hooray!

**Comment by biblicabeebli** [comment]
tamks

**Pull request merged**